wormhole-connect/src/utils/routes/routeAbstract.ts

```diff
@@ -9,9 +9,31 @@ import { ParsedMessage, ParsedRelayerMessage } from '../sdk';
 import { TransferDisplayData } from './types';
 import { BigNumber } from 'ethers';
 
+export type CCTPInfo = {
```

Comment:
Could this definition be moved to the `types.ts` file where `VaaInfo` is also defined?